lib/src/base/audio_waveforms_interface.dart

```diff
@@ -65,10 +65,10 @@ class AudioWaveformsInterface {
   }
 
   ///platform call to stop recording
-  Future<List<String?>?> stop() async {
-    final audioInfo =
+  Future<Map<String, dynamic>> stop() async {
+    Map<Object?, Object?> audioInfo =
```

Comment:
Can't we just use Map<String, dynamic>

Reply:
@ujas-m-simformsolutions No we can't because the invokeMethod returns result in Map<Object?, Object?>.